docs/6-developer-guide/1-contributor-guide.md

```diff
@@ -66,15 +64,15 @@ Our E2E tests can be found in the `/test` folder and follow the journey of someo
 
 ## Documentation
 
-In this section you'll find documentation on documentation! Pun absolutely intended :smile:
+PRs should also include any needed documentation updates.  The cobra CLI docs and JSON schema docs are updated via `zarf internal generate-cli-docs` and `zarf internal config-schema > zarf.schema.json`.
```

Comment:
I think they should run `.hooks/verify-zarf-schema.sh` instead so that the markdown gets created too.

Reply:
updated, thx.